### DISCUSSION TITLE
数据中心的容错时钟对时 · OSDI 2020 - 面向信仰编程 · /papers-sundial

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#636] [Announcements]
https://draveness.me/papers-sundial/

**Comment by dawncold** [comment]
图1是不是Server B在第一行？

---

*2021-01-31 UPDATES: 已修复*

**Comment by LehmanHe** [comment]
图一好像还是有误，Server A对应时间T，Server B对应时间T-1，图中反了
